### DISCUSSION TITLE
Advice for making a single page site with Pelican

### DISCUSSION THREAD

**Discussion by mar-ver** [getpelican/pelican#3416] [Q&A]
How would I go about creating a single page site with Pelican? 
This is for a portfolio, so my page would have sections with anchor pages: about, project 1, project 2, etc. 
Ideally I generate an index page from different markdown files, 1 for each section. 
So about.md, project1.md >> all their content goes in a single template. 
Two questions:
- How to disable Pelican creating 1 html file per markdown file? 
- How to display the mardown {{.content}} of each file. 

Or is Pelican not the right tool for that?

**Comment by pauloxnet** [comment]
Can you explain better what do you want to achieve with a complete code example?

**Comment by guillermodotn** [comment]
To create a single-page site with Pelican, you will need to create your own Pelican theme, as that is the only way to rearrange the contents.

If I’m correct in understanding what you’re trying to accomplish, take a look at the Pelican theme [resume](https://github.com/suheb/resume) for guidance. He uses variables declared in `pelicanconf.py` instead of the Markdown files.
